src/BloomExe/CollectionTab/LibraryView.cs

```diff
@@ -56,6 +61,19 @@ public partial class LibraryView :  UserControl, IBloomTabArea
 												});
 		}
 
+		private void BackgroundColorsForLinux() {
```

Comment:
Phil, I think this deserves at least a comment explaining why we need an image rather than just setting a background color on Linux, and why for this one view and not any other.